crypto/objects/objects.txt

```diff
@@ -33,6 +33,10 @@ member-body 840		: ISO-US		: ISO US Member Body
 ISO-US 10040		: X9-57			: X9.57
 X9-57 4			: X9cm			: X9.57 CM ?
 
+member-body 156         : ISO-CN        : ISO CN Member Body
+ISO-CN 10197            : oscca
```

Comment:
According to OID catalogues I can find, this is CCSTS rather than OSCCA...  but perhaps that's the same thing?

Reply:
CCSTS is supposed to be a committee belonging to OSCCA -- so it's better to call it CCSTS. (as shown in [@InfoHunter 's OID table](https://github.com/InfoHunter/misc/blob/master/GBT33560-2017-A1.md)